.github/workflows/maestro-changelog.yml

```diff
@@ -0,0 +1,65 @@
+name: Add changelog for Maestro bump
+on: pull_request
+
+jobs:
+  add-changelog:
+    name: Add changelog
+    runs-on: ubuntu-latest
+    if: github.actor == 'dotnet-maestro[bot]'
+
+    steps:
+    - name: 'Dump environment'
+      run: |
+        set -e
+        set -x
+        env | sort
```

Comment:
Is this desired or left over debugging aid?

Reply:
Removed!